controlplane/rosa/controllers/rosacontrolplane_controller.go

```diff
@@ -394,6 +407,41 @@ func (r *ROSAControlPlaneReconciler) reconcileDelete(ctx context.Context, rosaSc
 	return ctrl.Result{RequeueAfter: time.Second * 60}, nil
 }
 
+func (r *ROSAControlPlaneReconciler) reconcileClusterVersion(ctx context.Context, rosaScope *scope.ROSAControlPlaneScope, rosaClient *rosa.RosaClient, cluster *cmv1.Cluster) error {
+	version := rosaScope.ControlPlane.Spec.Version
+	if version == cluster.Version().RawID() {
```

Comment:
What does RawID() return?

Reply:
it returns the cluster version in the form of `x.y.z` like `4.14.5`
ID() would return something like `openshift-v4.14.5` or possibly `openshift-v4.14.5-<channel>`